databases/week1/db-week1.sql

```diff
@@ -0,0 +1,84 @@
+/*Find out how many tasks are in the task table*/
+SELECT 
+    COUNT(*) as 'Total Task'
```

Comment:
As per mentor review my HW.. you can als use  count(0) instead of count(*).Its give a same result but execute bit faster.... :)

Reply:
Thank you.